src/scripts/items/PokemonItem.ts

```diff
@@ -26,4 +26,5 @@ ItemList['Eevee'] = new PokemonItem(GameConstants.PokemonItemType.Eevee);
 ItemList['Porygon'] = new PokemonItem(GameConstants.PokemonItemType.Porygon);
 ItemList['Jynx'] = new PokemonItem(GameConstants.PokemonItemType.Jynx);
 ItemList['Mr. Mime'] = new PokemonItem(GameConstants.PokemonItemType.Mr_Mime);
-ItemList['Lickitung'] = new PokemonItem(GameConstants.PokemonItemType.Lickitung);
\ No newline at end of file
+ItemList['Lickitung'] = new PokemonItem(GameConstants.PokemonItemType.Lickitung);
+ItemList['Togepi'] = new PokemonItem(GameConstants.PokemonItemType.Togepi);
```

Comment:
It may be worth adding a new line to the end of the file so we have cleaner diffs in the future?
```suggestion
ItemList['Togepi'] = new PokemonItem(GameConstants.PokemonItemType.Togepi);

```